MovieEnthusiast.Api/Controllers/ApiControllerBase.cs

```diff
@@ -0,0 +1,13 @@
+﻿using MediatR;
+using Microsoft.AspNetCore.Mvc;
+
+namespace MovieEnthusiast.Api.Controllers;
+
+[ApiController]
+[Route("api/[controller]")]
```

Comment:
Can we think about our options for versioning here? Maybe it would require a new ticket altogether